src/Components/Publishing/Sections/Video.tsx

```diff
@@ -60,12 +76,22 @@ class VideoComponent extends React.Component<VideoProps, VideoState> {
         hidden: true
       }
     }
+    this.playVideo = this.playVideo.bind(this)
+ 
+  }
 
+  trackVideoClick() {
+    const trackParams = {
+      action: "Click",
+      label: "Play video",
+    }
+    this.props.tracking.trackEvent(trackParams)
   }
 
-  playVideo = () => {
+  playVideo() {
```

Comment:
Seeing as the data is static here, this is probably better served as a decorator...

````javascript
@track({ action: "Click", label: "Play video" })
playVideo() {
  ...
}
````

that's what makes it "declarative" as the lib authors say 🤷‍♂️